### ISSUE TITLE
[A11Y]: Accessibility audit round 1 issues

### ISSUE THREAD

**Issue by davidmenendez** [carbon-design-system/carbon-ai-chat#1128]
Reference issue #724 

Here are the issues that Ragu found on the initial walkthrough in order from highest to lowest severity;

### High severity

~~**SL No: 4 + 5 - Skip to main content link**~~

It looks like the hidden `skip to main content` link isn't working as intended for keyboard users and doesn't provide sufficient contrast when focused.

Additional clarification needed for it's intended purpose and functionality.

~~**SL No: 6 - Chat button toggle state**~~

In the nav bar, the `Open AI Chat` button doesn't work with keyboard. additionally, it needs to have the label update in accordance with the toggle state `{open / close} AI Chat`

~~**SL No: 7 - open resources button toggle state**~~

Similarly to the previous issue, the `Open resources button` needs to be keyboard operable and have the toggle state updated.

**SL No: 11 + 12 - custom chat keyboard navigation**

We need to get clarification on this. It seemed like during the initial meeting that there was some confusion on Ragu's team about how this functionality works. We need to get clarification on if this implementation is correct and what other keyboard / accessibility tool considers need to be made to get this to work properly

~~**SL No: 15 - scroll to bottom button missing label**~~

The issue says that the button is missing a label, but `aria-label` is being used on the button. Need clarification on what the issue is and how to fix it

### Medium severity

~~**SL No: 2 - missing H1**~~

The demo app needs to use an H1 somewhere at the top

~~**SL No: 3 + 8 - regions need to be redefined**~~

Ensure that elements are in the appropriate regions of main, aside (sidebar), and banner. also ensure that the option groups in the sidebar (page settings, chat config, etc) are properly semantically grouped. [Resources](https://www.w3.org/WAI/tutorials/forms/grouping/)

~~**SL No: 13 + 14 - disabled previous response button is focusable using  assistance**~~

the carbon `Dropdown` component uses `tab-index=0` even when disabled. this needs to be addressed so that the component is not tabbable when disabled

~~**SL No: 16 - Audio response missing title**~~

Need clarification on what the issue is and how to fix it

~~**SL No: 17 - Audio response requires a region**~~

There should be a defined region around the audio response / player

~~**SL No: 19 - Card response regions**~~

Need clarification on what the issue is and how to fix it

### Low severity

~~**SL No: 1 - Demo title attribute not descriptive enough**~~

Needs to be updated to say more than just demo

~~**SL No: 9 - Links inside open resource panel are focusable even when closed**~~

Just need to make sure the content isn't navigable when closed

~~**SL No: 10 - Send message label not descriptive enough**~~

Needs to be updated to say be more descriptive

~~**SL No: 18 - Card response links don't have proper labels**~~

> Control does not have proper  Label , Role and Description - Screen reader  users struggle to  understand the purpose /behavior.

Need clarification on what the issue is and how to fix it

~~**SL No: 20 - Card response alert actions**~~

The ok and cancel buttons in the first card response perform alerts when clicked. these labels should be updated to convey the functionality

**Comment by davidmenendez** [comment]
going to remove #1209 as a sub-issue since we've determined that, while we should still iterate on and improve on the hot key interaction, it's not a blocker. marking this as closed.
